Assets/Plugins/Source/Core/Utility/FileUtility.cs

```diff
@@ -32,6 +36,62 @@ namespace PlayEveryWare.EpicOnlineServices.Utility
     /// </summary>
     public static class FileUtility
     {
+        /// <summary>
+        /// Interval with which to update progress, in milliseconds
+        /// </summary>
+        private const int DefaultUpdateIntervalMS = 1500;
+
+        #region Data Structures 
+
+        /// <summary>
+        /// Stores information regarding a file copy operation.
+        /// </summary>
+        public struct CopyFileOperation
+        {
+            /// <summary>
+            /// The fully-qualified path of the source file.
+            /// </summary>
+            public string From;
```

Comment:
This might be pedantic, but instead of 'from' and 'to', can we use something like 'SourcePath' and 'DestinationPath'?

Reply:
Yeah, I think that's a reasonable thing to do. I _do_ prefer `From` and `To` to make things easier to read, but I think you're right that a more descriptive name is more appropriate.